AutomatedTesting/Gem/PythonTests/assetpipeline/asset_processor_tests/asset_relocator_tests.py

```diff
@@ -260,13 +260,13 @@ def test_WindowsMacPlatforms_AllowBrokenDependenciesWithoutMoveDelete_RelocatorW
         # Look for expected message inside the log and verify that no move or delete occurs in the log
         utils.validate_log_output(ap_batch_output, [expected_message], [unexpected_message])
 
-    @pytest.mark.skip(reason="https://github.com/o3de/o3de/issues/14514")
+    @pytest.mark.skipif(ly_test_tools.WINDOWS, reason="https://github.com/o3de/o3de/issues/14514")
```

Comment:
I like this, instead of marking skip because both platforms fail, you're leaving the skip for both so the separate reasons its failing are tracked.